packages/ui-components/src/components/NatAmountInput.jsx

```diff
@@ -1,6 +1,5 @@
 import { parseAsNat } from '../display/natValue/parseAsNat';
 import { stringifyNat } from '../display/natValue/stringifyNat';
-import { debounce } from '../helpers';
```

Comment:
should we remove the function itself?

Reply:
We may need it still. It's a good helper to have around.